gtk/FilterListModel.h

```diff
@@ -1,4 +1,4 @@
-// This file Copyright © 2023 Mnemosyne LLC.
+// This file Copyright © 2023-2023 Mnemosyne LLC.
```

Comment:
I don't think this was appropriate.

Reply:
My mistake, the script ended up adding an ending date. This will be fixed with a upcoming PR.